README.md

```diff
@@ -22,6 +26,7 @@ import FileIcon from 'react-file-icon';
 | `labelColor`      | string | undefined  | Color of label                    |
 | `labelTextColor`  | string | white      | Color of label text               |
 | `labelTextStyle`  | object | undefined  | Style of label text               |
+| `labelUppercase`  | bool   | false      | Displays the label in all caps    |
```

Comment:
Thoughts on setting `labelUppercase` to `true` by default? Keeps current behaviour and to me labels look a bit better in all caps 🤷‍♂️

Reply:
Resolved